torchx/schedulers/utils/ray_driver.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import logging
+import subprocess
+from ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, command, env):
+        self.command = command
+        self.env = env
+
+    def run_command(self):
+        return subprocess.run(self.command, env = self.env)
+
+def load_actor_json(filename : str) -> List[Dict]:
+    with open(filename) as f:
+        actor = json.load(f)
+        actor = json.loads(actor)
+        return actor
+
+if __name__ == "__main__":
```

Comment:
I suggest moving the rest of the code in this file into a function (e.g. `def _main()`). Otherwise, you won't be able to have proper tests for it.